portal-api/src/constants/email-template-ids.js

```diff
@@ -3,11 +3,13 @@ const BLOCKED = '82506983-cb85-4f33-b962-922b850be7ac';
 const UNBLOCKED = '44959d08-6389-4f27-a6be-2faae8bea711';
 const PASSWORD_UPDATE = '41235821-7e52-4d63-a773-fa147362c5f0';
 const UPDATE_STATUS = '718beb52-474e-4f34-a8d7-ab0e48cdffce';
+const SIGN_IN_LINK = '2eab0ad2-eb92-43a4-b04c-483c28a4da18';
```

Comment:
Thanks for the screenshot.

Are we going to need to do any extra styling work to make `placeholderSignInLink` display as a link when we implement it, or will it happen automatically if it is a URL?

Reply:
I've added a question for Adam to the Figma designs because I couldn't tell from the designs how we wanted the URL to be formatted

Reply:
Adam's decided that we want to have link text link so: [Click here to sign in to your account](url), which solves this problem. I've made this change to the template